client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -613,7 +613,7 @@ class WpcomChecklistComponent extends PureComponent {
 				buttonText={ translate( 'Launch site' ) }
 				completedTitle={ translate( 'You launched your site' ) }
 				description={ translate(
-					'Your site is private and only visible to you. Launch your site, when you are ready to make it public.'
+					'Your site is private and only visible to you. When you are ready, launch your site to make it public.'
```

Comment:
:disappointed: 20 existing translations will be lost with this change.

Reply:
The change in grammar is 👍 Ignore the i18n bot here.

My 2c, is to use a contraction to make it less formal: `When you're ready...`

I believe that's our general style. 

Talkin' to the kids in the kids' language. ♿️

Reply:
> use a contraction to make it less formal

yup yup!

Reply:
Sounds good - thanks for the note on the contraction/formality.